README.md

```diff
@@ -678,7 +678,7 @@ function quux () {}
 function quux2 () {
 
 }
-// Settings: {"jsdoc":{"matchingFileName":"/Users/brett/eslint-plugin-jsdoc/test/rules/data/test.js"}}
```

Comment:
Oh, looks like a local path snuck in here. Not sure what I can do about it in this PR, this has to be updated in upstream.

Reply:
Actually, I did the nasty thing and dropped out the commit with a force push. This way it's not in the diff at least and won't get in the way once the path reference is removed in upstream.

Reply:
Yeah, ESLint 6 requires absolute paths, and it seems the documentation generator resolves the path to the final result. For now, I can just rebase your PR and regenerate the docs on my machine to avoid your own path getting in there.

Reply:
Your force push solution is fine. Could you add this in parentheses though after `@template`: " (for Closure)"?

Reply:
or " (for Closure/TypeScript)"... It is not a standard jsdoc tag, so I think that should be called out for those who might otherwise try to search for the tag on https://jsdoc.app and not find it...

Reply:
Nevermind, I handled it (and changed the commit message to "docs:"). Thanks for the docs!